swift/WorkflowUI/Sources/Screen/Screen.swift

```diff
@@ -14,9 +14,18 @@
  * limitations under the License.
  */
 
+#if canImport(UIKit)
```

Comment:
This will effectively remove the Screen protocol for non UIKit targeting consumers. Just calling it out, we may not care right away but this may break external consumers if they build for MacOS for example.

Reply:
We could. But I doin’t think any of WorkflowUI makes sense outside of UIKit. If we wanted macOS support we’d probably want a separate module for that. It’s totally valid for a non-uikit related workflow to produce a rendering type that’s not a `Screen`.

Reply:
what about projects that want to use Workflow with SwiftUI?

Reply:
I know UIViewController's can be used in SwiftUI but UIKit technically isn't a requirement to develop in SwiftUI

Reply:
We have WorkflowSwiftUI, which doesn’t use WorkflowUI `Screen`